website_event_questions_template/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
```

Comment:
Better to remove here the e-mail, as in the header, and you have only one place where updating it:
```python
# Copyright 2017 Tecnativa - Sergio Teruel
```